Community/Tdarr_Plugin_x7ac_Remove_Closed_Captions.js

```diff
@@ -1,62 +1,54 @@
-/* eslint-disable */
 function details() {
   return {
-    id: "Tdarr_Plugin_x7ac_Remove_Closed_Captions",
-    Stage: "Pre-processing",
-    Name: "Remove closed captions",
-    Type: "Video",
-    Operation: "Remux",
+    id: 'Tdarr_Plugin_WS_Remove_Closed_Captions',
+    Stage: 'Pre-processing',
+    Name: 'Remove burned closed captions',
+    Type: 'Video',
+    Operation: 'Remux',
     Description:
-      "[Contains built-in filter] If detected, closed captions (XDS,608,708) will be removed.",
+      '[Contains built-in filter] If detected, closed captions (XDS,608,708) will be removed. Derived from x7ac Closed Captions plugin.',
```

Comment:
Nor is it derived from, it's updated :) so increment the version to `1.01` or something.

Reply:
Unintended change. Thanks!